src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="minutes minutes--speed-up"></div>
+      <div class="seconds seconds--speed-up"></div>
+      <div class="center"></div>
```

Comment:
Each stopwatch's minute and second div should have a text node or an element that can be used to display the time. Currently, these divs are empty, which doesn't follow the semantic structure of the content.